pkg/util/util.go

```diff
@@ -514,3 +515,29 @@ func ReadDirNoStat(dirname string) ([]string, error) {
 
 	return f.Readdirnames(-1)
 }
+
+func PrintByLine(w io.Writer, msg interface{}) {
```

Comment:
```suggestion
func PrintByLine(w io.Writer, errors interface{}) {
```
Could you please add a unit test here? :)